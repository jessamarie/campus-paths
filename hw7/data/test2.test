r
EMPAC
Field House Houston
r
76
52
r
76
Field House Houston
r
Polytechnic Residence Commons
Blitman Residence Commons
r
Blitman Residence Commons
35
r
131
131
q