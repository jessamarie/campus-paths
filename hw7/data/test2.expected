First building id/name, followed by Enter: Second building id/name, followed by Enter: Path from EMPAC to Field House Houston:
	Walk NorthEast to (Folsom Library)
	Walk North to (Lally Hall)
	Walk NorthEast to (Intersection 134)
	Walk NorthEast to (Intersection 133)
	Walk NorthEast to (Troy Building)
	Walk NorthEast to (Intersection 132)
	Walk NorthEast to (North Hall)
	Walk North to (H Building)
	Walk NorthEast to (Alumni House)
	Walk East to (Beman Park Firehouse)
	Walk East to (2021 Peoples Avenue)
	Walk NorthEast to (Intersection 112)
	Walk East to (Intersection 117)
	Walk East to (Intersection 111)
	Walk East to (Intersection 113)
	Walk East to (Field House Houston)
Total distance: 1473.118 pixel units.
First building id/name, followed by Enter: Second building id/name, followed by Enter: Path from EMPAC to Field House Houston:
	Walk NorthEast to (Folsom Library)
	Walk North to (Lally Hall)
	Walk NorthEast to (Intersection 134)
	Walk NorthEast to (Intersection 133)
	Walk NorthEast to (Troy Building)
	Walk NorthEast to (Intersection 132)
	Walk NorthEast to (North Hall)
	Walk North to (H Building)
	Walk NorthEast to (Alumni House)
	Walk East to (Beman Park Firehouse)
	Walk East to (2021 Peoples Avenue)
	Walk NorthEast to (Intersection 112)
	Walk East to (Intersection 117)
	Walk East to (Intersection 111)
	Walk East to (Intersection 113)
	Walk East to (Field House Houston)
Total distance: 1473.118 pixel units.
First building id/name, followed by Enter: Second building id/name, followed by Enter: Path from EMPAC to Field House Houston:
	Walk NorthEast to (Folsom Library)
	Walk North to (Lally Hall)
	Walk NorthEast to (Intersection 134)
	Walk NorthEast to (Intersection 133)
	Walk NorthEast to (Troy Building)
	Walk NorthEast to (Intersection 132)
	Walk NorthEast to (North Hall)
	Walk North to (H Building)
	Walk NorthEast to (Alumni House)
	Walk East to (Beman Park Firehouse)
	Walk East to (2021 Peoples Avenue)
	Walk NorthEast to (Intersection 112)
	Walk East to (Intersection 117)
	Walk East to (Intersection 111)
	Walk East to (Intersection 113)
	Walk East to (Field House Houston)
Total distance: 1473.118 pixel units.
First building id/name, followed by Enter: Second building id/name, followed by Enter: There is no path from Polytechnic Residence Commons to Blitman Residence Commons.
First building id/name, followed by Enter: Second building id/name, followed by Enter: Path from Blitman Residence Commons to Rensselaer Union:
	Walk East to (Louis Rubin Memorial Approach)
	Walk North to (Winslow Building)
	Walk NorthEast to (Intersection 94)
	Walk East to (Pittsburgh Building)
	Walk NorthEast to (Intersection 130)
	Walk East to (Intersection 131)
	Walk East to (Intersection 132)
	Walk East to (Intersection 108)
	Walk South to (Intersection 106)
	Walk East to (Rensselaer Union)
Total distance: 987.154 pixel units.
First building id/name, followed by Enter: Second building id/name, followed by Enter: Unknown building: [131]
